R/module_nested_tabs.R

```diff
@@ -1,8 +1,23 @@
 #' Calls all `modules`
 #'
-#' On the UI side each `teal_modules` is translated to a `tabsetPanel` and each `teal_module` is a
-#' `tabPanel`. Both, UI and server are called recursively so that each tab is a separate module and
-#' reflect nested structure of `modules` argument.
+#' Module call `modules` with `id` according to their location in the three.
+#' ### UI
+#' On the UI side drop-down is created listing buttons with labels of all `teal_module`(s).
+#' Buttons in a drop-down are grouped according to their group belonging.
+#' These UI components are created in a way to utilize navigation-bar functionality of shiny.
+#' To achieve this the buttons need to be placed inside a `ul` with a class `"nav shiny-tab-input"`.
```

Comment:
Not clear if ul or buttons need to Have this class 

```suggestion
#' To achieve this, the buttons need to be placed inside a `<ul class = "nav shiny-tab-input">`.
```